demo/protobuf_getting_started/update.sh

```diff
@@ -1,21 +1,20 @@
-#!/bin/bash
-
-clean() {
-  rm -rf public
-  rm -rf deps
-}
+#!/bin/bash -e
 
 if [ -z ${SRCDIR} ]; then
    DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
    SRCDIR=${DIR}/../..
 fi
 
-# Ensure that the .pb.go has been generated for the extensions
-# that are required.
-(cd ${SRCDIR}/proto/yext && SRCDIR=${SRCDIR} go generate)
-(cd ${SRCDIR}/proto/ywrapper && SRCDIR=${SRCDIR} go generate)
+GO_PATH="$(go env GOPATH)"
+
+if [ -z ${GO_PATH} ]; then
+  echo "no GOPATH defined!!!"
 
-clean
+  exit 1
+fi
+
+GO111MODULE=off go get -u github.com/openconfig/ygot || :
+GO111MODULE=off go get -u github.com/google/protobuf || :
```

Comment:
"Go 1.17 will ignore GO111MODULE": (https://blog.golang.org/go116-module-changes).

What's the reason why we should have this environment variable set?